model/task/task.go

```diff
@@ -1138,13 +1138,15 @@ func (t *Task) MergeNewTestResults() error {
 // MergeTestResultsBulk takes a slice of task structs and returns the slice with
 // test results populated. Note that the order may change. The second parameter
 // can be used to use a specific test result filtering query, otherwise all test
-// results for the passed in tasks will be merged
+// results for the passed in tasks will be merged. Display tasks will have
+// the execution task results merged
 func MergeTestResultsBulk(tasks []Task, query *db.Q) ([]Task, error) {
```

Comment:
Does modifying this function matter for any other queries that might depend on it?

Reply:
It does, but the current behavior is now correct - if there are display tasks that we are merging results for, we should look for their execution tasks